app-api/src/main/resources/liquibase-sender.xml

```diff
@@ -353,5 +353,38 @@
 
         <addNotNullConstraint tableName="sender_sync_request" columnName="status" columnDataType="VARCHAR(50)" />
     </changeSet>
+
+    <changeSet author="lpinto" id="20221013-1450">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="debezium_offset" />
+            </not>
+        </preConditions>
+        <comment>Adding debezium_offset table</comment>
+
+        <createTable tableName="debezium_offset">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" />
+            </column>
+            <column name="date_created" type="DATETIME(3)">
+                <constraints nullable="false" />
+            </column>
+            <column name="data" type="BLOB">
+                <constraints nullable="false" />
+            </column>
+            <column name="binlog_filename" type="VARCHAR(50)">
+                <constraints nullable="false" />
+            </column>
+            <column name="date_changed" type="TIMESTAMP(3)" />
+            <column name="enabled" type="BOOLEAN" defaultValueBoolean="true">
+                <constraints nullable="false" />
+            </column>
+        </createTable>
+    </changeSet>
+    
+    <changeSet author="lpinto" id="20221017-1100">
+        <comment>Migrating offset file to the database</comment>
```

Comment:
Don't you need a precondition?

Reply:
I do. Added accordingly.

Reply:
I don't think this is the correct precondition, the precondition needs to relate to what the changeset does, this changeset moves the offset file to the database, the precondition, I think it should check either that the table is empty

Reply:
Correct. I added that precondition. Thanks.